lib/watir-webdriver/browser.rb

```diff
@@ -10,6 +10,7 @@ class Browser
     include Waitable
 
     attr_reader :driver
+    attr_reader :checkers
```

Comment:
If we were starting from scratch, I would go with `AfterHooks` or `Checkers`. `AfterHooks` seems more accurate. However, since we have been using the term "checkers" for a while, I would be fine with continuing to use the term.

Perhaps we should rename everything to `AfterHooks`, but also make `checkers` an alias for `after_hooks`? This would allow us to use the better terminology while allowing existing users to stay with the current terminology.